.circleci/config.yml

```diff
@@ -24,6 +33,40 @@ references:
         tar -xz -C /tmp -f /tmp/docker-${DOCKER_CLIENT_VERSION:?}.tgz
         mkdir ~/bin && mv /tmp/docker/docker ~/bin
 
+  install_os_deps: &install_os_deps
+    run:
+      name: Install OS dependancies
+      command: |
+        sudo apt-get -qq -y install git curl \
```

Comment:
Could you please add to each of these install_* portions a comment with versioned link of where this setup is taken from? E.g. in this line https://github.com/aeternity/docker-builder/blob/0db5b300f251460e51e63ad5530a07896864f46b/Dockerfile#L4-L5

This would prevent Dev/DevOps people to amend this file as if it were the canonical copy.

Reply:
Added comment in dffc9bf